python-irclib.spec.in

```diff
@@ -26,9 +27,10 @@ python -c "import py_compile; py_compile.compile('ircbot.py')"
 
 %install
 [ "$RPM_BUILD_ROOT" != "/" ] && rm -rf $RPM_BUILD_ROOT
-%{__mkdir_p} $RPM_BUILD_ROOT/usr/lib/python/site-packages
-%{__install} -m 644 irclib.py* $RPM_BUILD_ROOT/usr/lib/python/site-packages
-%{__install} -m 644 ircbot.py* $RPM_BUILD_ROOT/usr/lib/python/site-packages
+SITE_PACKAGES_DIR=`python -c "from distutils.sysconfig import get_python_lib; print get_python_lib()"`
```

Comment:
This will check the site package directory, but it requires that the version of building the RPM is equal to the version in Thunder.